views/js/controller/runtime/testRunner.js

```diff
@@ -256,7 +265,36 @@ define(['jquery', 'lodash', 'spin', 'serviceApi/ServiceApi', 'serviceApi/UserInf
 			    }
 			}
 		},
+        /**
+         * Mark apropriate timer by warning colors and show feedback message
+         * @param {object} cst - Time constraint
```

Comment:
you can expand the documentation to precise what you expect into the object, for example: 

```javascript
/**
 * @param {Object} cst - Time constraint
 * @param {String} cst.className - desc
 * @param {Number} cst.seconds - desc
 * ...
```